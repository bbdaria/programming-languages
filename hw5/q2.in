use_module(library(yall)).
print("===TEST START===").

once(pythagorean(3, 4, 5)).
once(pythagorean(3, 3, 5)).

once(prime(4)).
once(prime(17)).

{L}/(
    findall(
        [X, Y],
        goldbach(X, Y, 20),
        L1
    ),
    msort(L1, L)
).
