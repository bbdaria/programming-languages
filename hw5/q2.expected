true.

true.

false.

false.

true.

L = [[3, 17], [7, 13], [13, 7], [17, 3]].


